components/server/ee/src/container-module.ts

```diff
@@ -132,4 +133,15 @@ export const productionEEContainerModule = new ContainerModule((bind, unbind, is
     bind(EntitlementServiceImpl).toSelf().inSingletonScope();
     rebind(EntitlementService).to(EntitlementServiceImpl).inSingletonScope();
     bind(BillingModes).to(BillingModesImpl).inSingletonScope();
+
+    // TODO(gpl) Remove as part of fixing https://github.com/gitpod-io/gitpod/issues/14129
+    rebind(UsageService)
+        .toDynamicValue((ctx) => {
+            const config = ctx.container.get<Config>(Config);
+            if (config.enablePayment) {
+                return ctx.container.get<UsageServiceImpl>(UsageServiceImpl);
+            }
+            return new UsageServiceMock();
```

Comment:
[nit] Let's call it a NoOpUsageService or something along the lines. Calling it a mock smells like this is test code.